src/ui/components/Dashboard/Dashboard.jsx

```diff
@@ -1,61 +1,151 @@
+  const [alertSeverity, setAlertSeverity] = useState('');
+  const [planDefFullUrl, setPlanDefFullUrl] = useState('');
+  const { data: planDefinitions } = usePlanDefinitions();
+
+  const handleChangePlanDefFullUrl = useCallback(
+    event => {
+      setPlanDefFullUrl(event.target.value);
     },
-    { key: 'logs', label: 'Logs', component: <Collections selectedCollection="logs" /> },
-    {
-      key: 'completedreports',
-      label: 'Completed Reports',
-      component: <Collections selectedCollection="completedreports" />
+    [setPlanDefFullUrl]
+  );
+
+  const handleChooseFile = useCallback(() => {
+    if (importFileRef?.current?.files?.[0]) setFileName(importFileRef.current.files[0].name);
+  }, [importFileRef, setFileName]);
+
+  const handleTrigger = useCallback(
+    event => {
+      event.preventDefault();
+      setIsLoading(true);
+      const files = importFileRef.current.files;
+      const reader = new global.FileReader();
+      reader.onload = e => {
+        const rawContent = e.target.result;
+        const resource = JSON.parse(rawContent);
+        axios
+          .post('/index/trigger', { planDefFullUrl, resource }, { withCredentials: true })
+          .then(() => {
+            setIsLoading(false);
+            setFileName('');
+            setPlanDefFullUrl('');
+            setAlert('Manual Report Triggered Successfully!', 'success');
+          })
+          .catch(err => {
+            console.error(err);
+            setAlert('Error Triggering Report Manually. Please Try Again.', 'error');
+          });
+      };
+      reader.readAsText(files[0]);
     },
-    {
-      key: 'reporting',
-      label: 'Reporting',
-      component: <Collections selectedCollection="reporting" />
+    [planDefFullUrl, setFileName, setIsLoading, setPlanDefFullUrl]
+  );
+
+  const handleCloseAlert = useCallback(() => {
+    setAlertMessage('');
+    setAlertSeverity('');
+  }, [setAlertMessage, setAlertSeverity]);
+
+  const setAlert = useCallback(
+    (message, severity) => {
+      setAlertMessage(message);
+      setAlertSeverity(severity);
     },
-    { key: 'errors', label: 'Errors', component: <Collections selectedCollection="errors" /> },
-    { key: 'requests', label: 'Requests', component: <Collections selectedCollection="requests" /> }
-  ];
-  const content = useMemo(() => {
-    return tabs.find(tab => {
-      return tab.key === contentKey;
-    });
-  }, [contentKey, tabs]);
+    [setAlertMessage, setAlertSeverity]
+  );
 
   return (
-    <div className={classes.container}>
-      <NavBar />
-      <Sidebar tabs={tabs} callback={setContentKey} selected={contentKey} />
-      <main className={classes.content}>
-        <div className={classes.spacer} />
-        {content ? content.component : 'Default'}
-      </main>
+    <div className={classes.dashboardContainer}>
+      <Paper elevation={3} className={classes.dashboardCard}>
+        <p>Manually Trigger Notification</p>
```

Comment:
Can the Manually Trigger Notification be a separate tab instead of on the main dashboard? I'm open to other opinions but it doesn't feel like it's important enough to be on the dashboard. (I do realize we don't have anything else on the dashboard)

Reply:
I'm open to moving it. It doesn't take up a ton of real estate though so I feel like having its own tab is overkill. Perhaps if we other "quick actions" those could all be a new tab?